README.md

```diff
@@ -281,47 +198,42 @@ SDC is composed of several pre-built components:
   a slightly customized build of vanilla SmartOS for SDC.
 - *Virtual machine images* for SDC services (e.g. imgapi, vmapi, adminui), which
   are provisioned as VMs at install time.
-- Agents, which are bundled into a [single
+- Agents bundled into a [single
   package](https://github.com/joyent/sdc-agents-installer)
-  that can then be installed into the global zone of Compute Nodes.
+  installed into the global zone of each compute node.
 
 Each component is built separately and then all are combined into CoaL and USB
 builds (see the preceding sections) via the [sdc-headnode
-repository](https://github.com/joyent/sdc-headnode). Built components are typically
-stored in a [Manta object store](https://github.com/joyent/manta), e.g.
-[Joyent's public Manta](https://www.joyent.com/products/manta), and pulled from
-there. For example, Joyent's core builds push to
+repository](https://github.com/joyent/sdc-headnode). The built components are
+typically stored in a [Manta object store](https://github.com/joyent/manta),
+e.g. [Joyent's public Manta](https://www.joyent.com/products/manta),
+and pulled from there. For example, Joyent's master builds push to
 `/Joyent_Dev/public/builds` in Joyent's public Manta in us-east-1
```

Comment:
That should stay "core", not "master". We do builds of branches other than "master".  Or if you meant "master" as other than the branch name, I suppose that is confusing overloading of "master". :)